src/webpack/loaders/devSiteEntry.js

```diff
@@ -0,0 +1,83 @@
+const lodashTemplate = require('lodash.template');
+const { getOptions } = require('loader-utils');
+
+const generateNavigationConfig = require('../loaderUtils/generateNavigationConfig');
+
+/**
+ * Generate the example with the supplied file.
+ * Don't use an arrow function
+ */
+const loader = async function loader(template) {
+  const callback = this.async();
+
+  const imports = {};
+  let index = 0;
+
+  const addImport = (path) => {
+    let ident = imports[path];
+    if (ident) {
+      return ident;
+    }
+    ident = `import${index}`;
+    index += 1;
+    imports[path] = ident;
+    return ident;
+  };
+
+  const {
+    entryPath,
+    siteConfig,
+    basename,
+    resolveExtensions,
+    sites,
+    contentDirectory,
+    isLernaMonoRepo,
+  } = getOptions(this);
+
+  const extensionItems = (siteConfig.extensionItems || []).map((ext) => ({
+    key: ext.key,
+    text: ext.text,
+    icon: addImport(ext.iconPath),
+    modal: addImport(ext.modalFilePath),
+  }));
+
+  const {
+    contentImports,
+    navigationConfig,
+    routesMap,
+    pageConfig,
+  } = generateNavigationConfig({
+    siteConfig,
+    resolveExtensions,
+    mode: this.mode,
+    verbose: false,
```

Comment:
Will this always be false? Should verbose be removed?

Reply:
yep thats dead code